Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "MIT OR Apache-2.0"
 name = "cortex-m-rt"
 readme = "README.md"
 repository = "https://github.com/japaric/cortex-m-rt"
```

Comment:
This should probably be changed to https://github.com/rust-embedded/cortex-m-rt at some point?